src/main/java/com/rflpazini/badabin/router/HelloRouter.java

```diff
@@ -15,6 +15,7 @@
   @Bean
   public RouterFunction<ServerResponse> helloRoute(HelloHandler helloHandler) {
     return RouterFunctions.route(
-            GET("/badabin"), helloHandler::sayHello);
+            GET("/badabin"), helloHandler::sayHello)
+      .andRoute(GET("/ab"), helloHandler::saySomething);
```

Comment:
O fulano, isso aqui você pode fazer assim 
```suggestion
      .andRoute(GET("/newEndpoint"), helloHandler::saySomething);
```